workers/data_refinery_workers/processors/create_compendia.py

```diff
@@ -207,8 +211,11 @@ def _perform_imputation(job_context: Dict) -> Dict:
     # Perform a full outer join of microarray_matrix and
     # log2_rnaseq_matrix; combined_matrix
     if log2_rnaseq_matrix is not None:
+        # Use how='right' because we've filtered some gene identifiers
```

Comment:
I'm not sure we want to keep those out of the full matrix. I suspect they will be dropped in the later steps but I don't know for sure that we could guarantee that. Is there a major downside to keeping this from `outer`? I think it should not reallocate much, since the full set is given by the left matrix.

Reply:
I would think that would be fine. I just made an assumption that a gene that had no representation in the RNA-Seq data would be one we we'd wanna keep out of the final matrix. I'll change this back!

Reply:
I'd guess it'll get filtered later and this is almost certainly a safe change, but if we don't need it - my preference is to avoid it until we can examine whether or not it makes a difference. 👍